File: bronze/2609/python/main.py

if (__name__ == "__main__"):
    a, b = map(int, input().split())
    tempA, tempB = (a, b)
    
    while (tempB):
        tempA, tempB = (tempB, tempA % tempB)
    print(tempA)
    print(a * b // tempA)

File: bronze/2609/c++/main.cpp
#include <iostream>

int main()
{
    int a, b;
    std::cin >> a >> b;

    int temp;
    int tempA = a, tempB = b;

    while (tempB) {
        temp = tempB;
        tempB = tempA % tempB;
        tempA = temp;
    }

    std::cout << tempA << '\n';
    std::cout << a * b / tempA;

    return 0;
}